### PR TITLE
Release 100.12.0

### PR DESCRIPTION
**Pull request by puneet-pdx**
Merges `Develop` into `Master`

### CODE REVIEW

File: gradle/script/versions.gradle [hunk 1]
```diff
@@ -25,20 +25,20 @@ ext {
         multiDex = "2.0.1"
         testRunner = "1.3.0"
         arCore = "1.17.1"
-        lifecycle = "2.2.0"
-        appcompat = "1.2.0"
-        material = "1.2.1"
+        lifecycle = "2.3.1"
```
Comment (sorenoid) [inline gradle/script/versions.gradle:28]: is this still in use?

**Review by sorenoid — APPROVED** [review]
👍

**Review by gunt0001 — APPROVED** [review]

File: gradle/script/versions.gradle (hunk 1 above)
Comment (puneet-pdx) [reply gradle/script/versions.gradle:28]: yes

**Review by eri9000 — APPROVED** [review]
👍🏼

**Pull request merged**